src/sync.js

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+import Cache from "./Cache.js";
+import Logger from "js-logger";
+
+const sync = (async () => {
+    // http://time.google.com:80 doesn't actually give us an NTP response, instead it returns
+    // a 302 redirected response. However, it does contain a `date` header which we can use.
+    try {
+        const response = await axios.get("http://time.google.com", {
+            maxRedirects: 0,
+            
+            // By default a status of 302 is considered an erring status
+            validateStatus: function (status) {
+                return status < 500;
+            },
+        });
+
+        const currentTime = Math.round(Date.now() / 1000);
+
+        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date
+        // strongly discourages us using `Date.parse()`, but I'm not sure what we should replace
+        // it with without adding any new deps.
```

Comment:
The only other alternative (without resorting to moment or date-fns) I see is to [manually parse](https://jsfiddle.net/willparks/ber8f0hc/32/) the Date header, but it is a bit verbose. Otherwise, I'm not sure what else can be done here either.

Reply:
I would really like to avoid hand rolling parsing tbh. If we end up having trouble with `Date.parse()` we might end up hand rolling it or adding a library for it, but for the time being I think continuing to use `Date.parse()` is fine?

Reply:
I completely agree, `Date.parse()` is working perfectly fine - no sense in reinventing the wheel :)